symfony/framework-bundle/3.3/src/Kernel.php

```diff
@@ -52,10 +52,19 @@ protected function configureContainer(ContainerBuilder $container, LoaderInterfa
 
     protected function configureRoutes(RouteCollectionBuilder $routes)
     {
-        $confDir = $this->getProjectDir().'/config';
-
+        $confDir = $this->getConfigDir();
         $routes->import($confDir.'/{routes}/*'.self::CONFIG_EXTS, '/', 'glob');
         $routes->import($confDir.'/{routes}/'.$this->environment.'/**/*'.self::CONFIG_EXTS, '/', 'glob');
         $routes->import($confDir.'/{routes}'.self::CONFIG_EXTS, '/', 'glob');
     }
+
+    private function getConfigDir()
+    {
+        return $this->getProjectDir().'/config';
+    }
+
+    private function getBundlesFile()
```

Comment:
IMO, this method should not exist at all. Changing its implementation must never happen, as Flex will keep using `%CONFIG_DIR%/bundles.php`. So I would keep `$this->getConfigDir().'/bundles.php'` inline in both places